test/foundry/apps.t.sol

```diff
@@ -749,4 +750,187 @@ contract FleekTest is Test {
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+        fleekContract.increaseAccessPointScore("https://fleek_cloned.xyz");
+        fleekContract.decreaseAccessPointScore("https://fleek_cloned.xyz");
+        assertEq(
+            fleekContract.getAccessPointJSON("https://fleek_cloned.xyz"),
+            '{"tokenId":0,"score":0,"nameVerified":false,"contentVerified":false,"owner":"0x7fa9385be102ac3eac297483dd6233d62b3e1496"}'
+        );
+    }
+
+    function testAppAccessPoints() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+
+        string[] memory accessPointList = fleekContract.appAccessPoints(mint);
+        assertEq(accessPointList[0], "https://fleek_cloned.xyz");
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned_2.xyz");
+
+        accessPointList = fleekContract.appAccessPoints(mint);
+        assertEq(accessPointList[1], "https://fleek_cloned_2.xyz");
+    }
+
+    function testFailSetAccessPointNameVerifyWithUnknownIdentity() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+
+        vm.prank(address(0xb4c79daB8f259C7Aee6E5b2Aa729821864227e84));
+
+        fleekContract.setAccessPointNameVerify("https://fleek_cloned.xyz", true);
+    }
+
+    function testFailSetAccessPointContentVerifyWithUnknownIdentity() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+
+        vm.prank(address(0xb4c79daB8f259C7Aee6E5b2Aa729821864227e84));
+
+        fleekContract.setAccessPointContentVerify("https://fleek_cloned.xyz", true);
+    }
```

Comment:
No assertions

Reply:
No need because fleek access control fails the call:
`FleekAccessControl: must have token role`